### PR TITLE
PRESIDECMS-1985 Add webp to core asset manager file types

### PR DESCRIPTION
**Pull request by teonator**
Add webp i18n. Update file type picker to handle unknown file type i18n instead of showing NOT FOUND.

### CODE REVIEW

File: system/handlers/formcontrols/FileTypePicker.cfc [hunk 1]
```diff
@@ -16,7 +16,7 @@ component {
 			for( var type in assetTypes[ assetTypeGroup ] ){
 				types.append( {
 					  value = type
-					, label = translateResource( "filetypes:#type#.picker.label" )
+					, label = translateResource( "filetypes:#type#.picker.label", translateResource( "filetypes:#assetTypeGroup#.picker.text", "Unknown" ) & ": #type#" )
```
Comment (DominicWatson) [inline system/handlers/formcontrols/FileTypePicker.cfc]: If we are going to do defaults here, we need to default everything. So two extra i18n URIs: 

```properties
unknown.group.type=Unknown
default.type.label={1}: {2}
```

Then, here:

```cfc
groupType = translateResource( "filetypes:#assetTypeGroup#.picker.text", translateResource( "filetypes:unknown.group.type" ) );
default = translateResource( uri="default.type.label", data=[ groupType, type ]  );
// ...
label = translateResource( "filetypes:#type#.picker.label", default )
```

File: system/i18n/filetypes.properties [hunk 2]
```diff
@@ -36,7 +38,9 @@ ppam.picker.label=Document: ppam
 pptm.picker.label=Document: pptm
 potm.picker.label=Document: potm
 ppsm.picker.label=Document: ppsm
+webp.picker.label=Document: webp
```
Comment (DominicWatson) [inline system/i18n/filetypes.properties]: Should `webp` not be video?

Also, this should be added to `/system/config/Config.cfc$_getConfiguredFileTypes()`.

**Review by DominicWatson — COMMENTED** [review]
Hi @teonator - thanks for this, looking good. Could you check the review items and I'll merge when done.

File: system/handlers/formcontrols/FileTypePicker.cfc (hunk 1 above)
Comment (teonator) [reply system/handlers/formcontrols/FileTypePicker.cfc]: Updated with additional suggested i18n and related changes.

File: system/i18n/filetypes.properties (hunk 2 above)
Comment (teonator) [reply system/i18n/filetypes.properties]: `webp` is an image. https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types.
Updated i18n and added it into config.

Comment (teonator) [reply system/i18n/filetypes.properties]: I also found and fixed typo `serveAsAttachmemnt` of svg mime type in config.

**Comment by DominicWatson** [comment]
Perfect, looking good. Thanks for the typo fix too ;)

**Comment by DominicWatson** [comment]
I created a new separate ticket for this (as not really related to form builder): https://presidecms.atlassian.net/browse/PRESIDECMS-1985

I've merged this into 10.13.0 upcoming release ready for internal QA.

**Pull request merged**